executor/infoschema_reader.go

```diff
@@ -1482,103 +1480,6 @@ func keyColumnUsageInTable(schema *model.DBInfo, table *model.TableInfo) [][]typ
 	return rows
 }
 
-func (e *memtableRetriever) setDataForTiKVRegionStatus(ctx sessionctx.Context) (err error) {
```

Comment:
How would tidb read data for tikvRegionStatus if the retriever is removed?

Reply:
> How would tidb read data for tikvRegionStatus if the retriever is removed?

I move it to memtablereader,as this pull request show https://github.com/pingcap/tidb/pull/30648/files